wranglerjs/index.js

```diff
@@ -54,6 +54,18 @@ function filterByExtension(ext) {
   }
   config.target = "webworker";
 
+  // The worker runtime will set the name of the script to `worker.js`,
+  // regardless of what's specified in the sourcemap.
+  // We can tell webpack to name the generated worker by configuring the output.
+  // It's also safe to force that configuration because it mirrors what the
+  // runtime does.
+  // https://github.com/cloudflare/wrangler/issues/681
+  if (config.output === undefined) {
+    config.output = {};
+  }
```

Comment:
what do you think about adding some logic here to provide this info to users? i'm thinking a println with an emoji::INFO should do the trick: maybe something like "webpack's output filename is being renamed to `worker.js` because of requirements from the Workers runtime"

Reply:
I'm not so an emoji person but I will give it a try

Reply:
@EverlastingBugstopper can you file an issue for this, i'm gonna merge this :)